src/sniffer.c

```diff
@@ -4487,7 +4495,11 @@ static void RemoveStaleSessions(void)
         session = SessionTable[i];
         while (session) {
             SnifferSession* next = session->next;
+        #if !defined(NO_ASN) && !defined(NO_ASN_TIME)
```

Comment:
The sniffer can require ASN time support. Please revert and have it always call `wc_Time`.